### ISSUE TITLE
`GenericFormattingRegistry` missing from pint 0.24

### ISSUE THREAD

**Issue by lgrcia** [dfm/jpu#15]
From a fresh install importing `jpu` gives
```
  File "<string>", line 1, in <module>
  File "/Users/lgarcia/mambaforge/envs/test_jpu/lib/python3.10/site-packages/jpu/__init__.py", line 10, in <module>
    from jpu import monkey, numpy
  File "/Users/lgarcia/mambaforge/envs/test_jpu/lib/python3.10/site-packages/jpu/monkey.py", line 7, in <module>
    from jpu.registry import UnitRegistry
  File "/Users/lgarcia/mambaforge/envs/test_jpu/lib/python3.10/site-packages/jpu/registry.py", line 29, in <module>
    facets.GenericFormattingRegistry[facets.QuantityT, facets.UnitT],
AttributeError: module 'pint.facets' has no attribute 'GenericFormattingRegistry'. Did you mean: 'GenericPlainRegistry'?
```

It seems `GenericFormattingRegistry` was removed from `pint.facets` in [this commit](https://github.com/hgrecco/pint/commit/17c2143386349fc503d6b01b4b84d8db85ed06df), but I don't know enough of pint to understand if it's a bug or if this object is deprecated.  

For now downgrading to `pint==0.23` works.

**Comment by dfm** [comment]
Should be fixed by #16

**Comment by dfm** [comment]
New version released: https://pypi.org/project/jpu/0.0.4/
